scout/parse/variant/frequency.py

```diff
@@ -20,7 +20,8 @@ def parse_frequencies(variant, transcripts):
     exac_keys = ['EXACAF']
     exac_max_keys = ['ExAC_MAX_AF', 'EXAC_MAX_AF']
 
-    gnomad_keys = ['GNOMADAF', 'GNOMAD_AF']
+    # Gnomad have both snv and sv frequencies
+    gnomad_keys = ['GNOMADAF', 'GNOMAD_AF', 'gnomad_svAF']
```

Comment:
Could you check that this is being displayed in the SV variants view?

Reply:
I haven't tried with such a VCF. We could maybe pull the branch on stage and see if uploads a test case with these keys.

Reply:
Is anyone using hasta stage?